proxy/files/bin/proxy-start.sh

```diff
@@ -21,7 +22,18 @@ if [ "${INTERNAL_GW_URL:-X}" = 'X' ]; then
   exit 1
 fi
 
-erb /etc/nginx/conf.d/evaka-nginx.conf.template > /etc/nginx/conf.d/evaka-nginx.conf
+if test -n "$DEPLOYMENT_BUCKET"; then
+  s3download "$DEPLOYMENT_BUCKET" "nginx-proxy" /etc/nginx/
+fi
+
+for template in `ls -v /etc/nginx/conf.d/*.template`; do
```

Comment:
Supernit: I'd prefer `$(...)` over backticks just to keep a consistent style (and avoid unnecessary complaints from shellcheck, https://github.com/koalaman/shellcheck/wiki/SC2006).

Also, prefer globs over `ls` output, see [rationale](https://github.com/koalaman/shellcheck/wiki/SC2045).